java/src/main/java/org/eclipse/ditto/client/management/CommonManagement.java

```diff
@@ -326,6 +328,7 @@
      * {@code "thingId"}.
      * @throws org.eclipse.ditto.model.base.exceptions.DittoJsonException if {@code thing} cannot be parsed to a {@link
      * Thing}.
+     * @since 1.0.0
```

Comment:
was this already in 1.0.0?

Reply:
The interface method `put(JsonObject thing, Option<?>... options)` was already there from the beginning. So imo `@since 1.0.0` must be right